src/compas/geometry/frame.py

```diff
@@ -604,6 +604,18 @@ def to_transformation(self):
     # Methods
     # ==========================================================================
 
+    def invert(self):
```

Comment:
not sure but should this be maybe called `flip`?

Reply:
in `Vector` it is also `invert`, but we can add an alias in both perhaps